app/views/layouts/_footer.html.erb

```diff
@@ -3,7 +3,14 @@
     <div class="inner-footer-container">
       <a href="/">Home</a> <a href="/about">About</a> <a href="/privacy">Privacy Policy</a>
       <a href="/terms">Terms of Use</a> <a href="/contact">Contact</a> <a href="/code-of-conduct">Code of Conduct</a><br/>
-      <%= ApplicationConfig["COMMUNITY_NAME"] %> Community copyright 2016 - <%= Time.current.year %>&nbsp; 🔥
+      <%= ApplicationConfig["COMMUNITY_NAME"] %> Community copyright <%= "2016 -" if base_community? %> <%= Time.current.year %>&nbsp; 🔥
```

Comment:
Does it make sense to add a todo somewhere that `ApplicationConfig["COPYRIGHT_START_YEAR"]` has to be defined in the future?